src/coldfront_plugin_api/scim_v2/auth_middleware.py

```diff
@@ -0,0 +1,17 @@
+import os
+import logging
+
+from django_scim.middleware import SCIMAuthCheckMiddleware
+from mozilla_django_oidc.contrib.drf import OIDCAuthentication
+
+logger = logging.getLogger(__name__)
+
+
+class SCIMColdfrontAuthCheckMiddleware(SCIMAuthCheckMiddleware):
+    def process_request(self, request):
+        if not request.user or not request.user.is_authenticated:
+            # PLUGIN_AUTH_OIDC implies DRF OIDC backend is configured
+            if os.getenv("PLUGIN_AUTH_OIDC") == "True":
+                oidc_auth_obj = OIDCAuthentication()
+                request.user = oidc_auth_obj.authenticate(request)
```

Comment:
The above 2 lines can be merged into line. No need to assign the instance to a variable if you're only using it once.

Also, I'm confused. `authenticate()` appears to [return a tuple](https://github.com/mozilla/mozilla-django-oidc/blob/2c2334fdc9b2fc72a492b5f0e990b4c30de68363/mozilla_django_oidc/contrib/drf.py#L105) whereas you are assigning only 1 return value. Is there something I'm missing?

Reply:
You are right, `authenticate()` does return a tuple. I noticed this too during testing, but I guess I forgot to index the tuple.

Reply:
Do you have a timeline for when you expect to be done with the testing?

Reply:
I will have the new testcases submitted by the end of this week.